packages/diffhtml/lib/util/config.js

```diff
@@ -59,7 +59,7 @@ export default function getConfig(name, defaultValue, type = typeof defaultValue
   const hasSearchParams = typeof URLSearchParams !== 'undefined';
   const hasLocation = typeof location !== 'undefined';
   const useSearchParams = hasSearchParams && hasLocation;
-  const useEnv = process.env;
+  const useEnv = typeof process !== 'undefined' && process.env;
```

Comment:
`process` comes from `util/process.js`, which is already guarded against undefined:

https://github.com/tbranyen/diffhtml/blob/master/packages/diffhtml/lib/util/process.js#L3-L6